index.md

```diff
@@ -21,7 +21,7 @@ CMOR output has the following characteristics:
 
 * For metadata, different MIPs may have different requirements, but these are accommodated by CMOR, within the constraints of the CF convention and as specified in the MIP tables (e.g. [CMIP6 MIP tables](https://github.com/PCMDI/cmip6-cmor-tables)).
 
-* CMOR can rely on NetCDF4 [See unidata web page](http://www.unidata.ucar.edu/software/netcdf) to write the output files and can take advantage of its compression and chunking capabilities. In that case, compression is controlled with the MIP tables using the shuffle, deflate and deflate_level attributes, default values are respectively 0, 0 and 0(disable). It is worth noting that even when using NetCDF4, CMOR3 still produces NETCDF4 CLASSIC formatted output. This allows the file generated to be readable by any application that can read NetCDF3 provided they are re-linked against NetCDF4.  When using the NetCDF4 library it is also still possible to write files that can be read through the NetCDF3 library by adding "_3" to the appropriate cmor_setup argument (see below). Note: CMOR3 **NOW** output NetCDF4 files by default.  For CMIP6, the NetCDF4/NC_CLASSIC_Model mode is used (and chunking is **NOW** invoked... shuffle and deflation can be invoke on-demand by setting flags in the table. [example](https://github.com/PCMDI/cmor/blob/master/Test/speed_test_table_A#L1691-L1693)).
+* CMOR can rely on NetCDF4 [See unidata web page](http://www.unidata.ucar.edu/software/netcdf) to write the output files and can take advantage of its compression and chunking capabilities. In that case, compression is controlled with the MIP tables using the shuffle, deflate and deflate_level attributes, default values are respectively 0, 0 and 0(disable). It is worth noting that even when using NetCDF4, CMOR3 still produces NETCDF4 CLASSIC formatted output. This allows the file generated to be readable by any application that can read NetCDF3 provided they are re-linked against NetCDF4.  When using the NetCDF4 library it is also still possible to write files that can be read through the NetCDF3 library by adding "_3" to the appropriate cmor_setup argument (see below). Note: CMOR3 **NOW** output NetCDF4 files by default.  For CMIP6, the NetCDF4/NC_CLASSIC_Model mode is used (and chunking is **NOW** invoked... shuffle and deflation can be invoke on-demand by setting flags in the table. [example](https://github.com/PCMDI/cmor/blob/main/Test/speed_test_table_A#L1691-L1693)).
```

Comment:
Compression isn't controlled in the MIP tables - rather this would be the user input, no?

Note: CMOR3 Now output **s** NetCDF4 file by default (missing s)

..invoke **d** on demand.. (missing d)

Reply:
I'll remove the sentence `In that case, compression is controlled with the MIP tables using the shuffle, deflate and deflate_level attributes, default values are respectively 0, 0 and 0(disable).`.  Another thing to note is that chunking is not really controlled by the user.  CMOR will set chunking for variables but using netCDF4's default chunk size.